src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -352,7 +363,8 @@ object ScalafixPlugin extends AutoPlugin {
         loadedRules = () => scalafixInterfaceProvider.value().availableRules(),
         terminalWidth = Some(JLineAccess.terminalWidth),
         allowedTargetFilesPrefixes =
-          (scalafix / unmanagedSourceDirectories).value.map(_.toPath)
+          (scalafix / unmanagedSourceDirectories).value.map(_.toPath),
+        jgitCompletion = (ThisBuild / scalafixJGitCompletion).value
```

Comment:
This shouldn't matter for most cases, but for the sake of potential extensibility we could let potential project-level overrides
```suggestion
        jgitCompletion = scalafixJGitCompletion.value
```